src/components/studies/app-design/AppDesign.tsx

```diff
@@ -640,6 +721,10 @@ const AppDesign: React.FunctionComponent<
                 }}
                 studyTitle={study.name || ''}
                 studySummaryBody={study.details || ''}
+                studyTitleHasError={showError && errorState.studyTitleHasError}
```

Comment:
here and  below there is a bit of repetition.. What about making a function that takes the errorState property you are looking for? 
the fn would look something like this
```
function hasError(errorType: [your type can be a keyof of your error type]) {
return showError && errorState[errorType'
}
```
then you would just all studyTitleHasError={hasError('studyTitleHasError')}